integration-tests/tests/api/policy/policy-check.spec.ts

```diff
@@ -0,0 +1,274 @@
+import { prepareProject } from 'testkit/registry-models';
+import { ProjectType, RuleInstanceSeverityLevel, SchemaPolicyInput } from '@app/gql/graphql';
+import { createCLI } from '../../../testkit/cli';
+
+export const createPolicy = (level: RuleInstanceSeverityLevel): SchemaPolicyInput => ({
+  rules: [
+    {
+      ruleId: 'require-description',
+      severity: level,
+      configuration: {
+        types: true,
+      },
+    },
+  ],
+});
+
+describe('Schema policy checks', () => {
+  describe('model: composite', () => {
+    it('Fedearation project with policy with only warnings, should check only the part that was changed', async () => {
+      const { tokens, policy } = await prepareProject(ProjectType.Federation);
+      await policy.setOrganizationSchemaPolicy(
+        createPolicy(RuleInstanceSeverityLevel.Warning),
+        true,
+      );
+      const cli = await createCLI(tokens.registry);
+
+      await cli.publish({
+        sdl: /* GraphQL */ `
+          type Product @key(fields: "id") {
+            id: ID!
+            title: String
+            url: String
+          }
+
+          extend type Query {
+            product(id: ID!): Product
+          }
+        `,
+        serviceName: 'products',
+        serviceUrl: 'https://api.com/products',
+        expect: 'latest-composable',
+      });
+
+      await cli.publish({
+        sdl: /* GraphQL */ `
+          type User @key(fields: "id") {
+            id: ID!
+            name: String!
+          }
+
+          extend type Query {
+            user(id: ID!): User
+          }
+        `,
+        serviceName: 'users',
+        serviceUrl: 'https://api.com/users',
+        expect: 'latest-composable',
+      });
+
+      const message = await cli.check({
+        sdl: /* GraphQL */ `
+          type User @key(fields: "id") {
+            id: ID!
+            name: String!
+          }
+
+          extend type Query {
+            user(id: ID!): User
+          }
+        `,
+        serviceName: 'users',
+        expect: 'approved',
+      });
+
+      expect(message).toContain(`Detected 1 warning`);
+      expect(message).toContain(
+        'Description is required for `type User`. (line: 1, column: 6, source: policy-require-description)',
```

Comment:
I'm not sure if we should include `line: 1, column: 6` or not :(

Reply:
yeah, I was wondering the same. I think I will drop it. @B2o5T is working on adding better messages for the eslint rules so we can drop the col/line thing.

Reply:
done